R/get_vespadb_export_s3.R

```diff
@@ -0,0 +1,51 @@
+#' get_vespadb_export_s3
```

Comment:
I suggest we rename this to `get_vespadb_export` since the original doesn't work anymore and wasn't ported to this package.